chain/owasm/execute_test.go

```diff
@@ -13,7 +13,7 @@ func TestExecuteCanCallEnv(t *testing.T) {
 	if err != nil {
 		panic(err)
 	}
-	result, gasUsed, err := Execute(NewMockExecutionEnvironment(), code, "execute", []byte{}, 10000)
+	result, gasUsed, err := Execute(&mockExecutionEnvironment{}, code, "execute", []byte{}, 10000)
 	if err != nil {
```

Comment:
Use `require.NotNil(t, err)` instead if and panic

Reply:
Thanks! Should be `require.Nil` though.